Automattic/wp-calypso
# Update Woo Express plans grid

We're updating the Woo Express plans grid to only have two columns: one for the Essential plan, and one for Performance plan, removing the Plus plan altogether. Beneath these plans, we’d have a CTA for high-volume stores that can benefit from our Woo Enterprise offering.

See the post here for reference: peapX7-2ct-p2

Related to wc-calypso-bridge#1108

## Proposed Changes

* Remove Plus from Woo Express plans grid
* Add a Woo Enterprise banner at bottom of page
* Update font stack for the Comparison grid to use sans-serif instead of Recoleta

## Testing Instructions

* Build this branch
* View the expired plan URL which works for any site: http://calypso.localhost:3000/plans/my-plan/trial-expired/[ANY WP.COM URL] 
* Additionally, you can sign up for a Woo Express trial site at https://wordpress.com/start/
* Once the trial site is setup, view the [plans page](https://wordpress.com/plans/) for that site (Upgrades > Plans)
* There should only be two visible plans, and a banner for Enterprise ecommerce at the bottom of the plans grid
* View the pages at different viewports and make sure the Enterprise banner behaves responsively


**Before**
![image](https://github.com/Automattic/wp-calypso/assets/4297811/64657c6b-856c-4dee-833e-92c7181bb8be)

**After**
![image](https://github.com/Automattic/wp-calypso/assets/4297811/fc7329e2-b8ea-47d6-b7fd-72dba0c29eb6)


## Pre-merge Checklist
